neural_compressor/torch/quantization/config.py

```diff
@@ -47,98 +44,103 @@ class OperatorConfig(NamedTuple):
 
 
 ######################## RNT Config ###############################
-
-
 @register_config(framework_name=FRAMEWORK_NAME, algo_name=RTN, priority=PRIORITY_RTN)
 class RTNConfig(BaseConfig):
     """Config class for round-to-nearest weight-only quantization."""
 
-    supported_configs: List[OperatorConfig] = []
+    name = RTN
     params_list = [
-        "weight_dtype",
-        "weight_bits",
-        "weight_group_size",
-        "weight_sym",
-        "act_dtype",
-        "enable_full_range",
-        "enable_mse_search",
-        "group_dim",
-        "return_int",
+        "dtype",
+        "bits",
+        "group_size",
+        "use_sym",
+        "use_full_range",
+        "use_mse_search",
+        "use_layer_wise",
+        "export_compressed_model",
+        "use_double_quant",
         "double_quant_dtype",
```

Comment:
please update double quant configs in neural_compressor.torch.utils.constans.py.

Reply:
OK